src/jvm/main/org/jetbrains/kotlinx/lincheck/runner/TestThreadExecution.java

```diff
@@ -42,6 +42,7 @@
     public int[][] clocks; // for HBClock
     public volatile int curClock;
     public boolean useClocks;
+    public boolean useClocksOnce = false;
```

Comment:
When a system crash occurred, we want to be sure that next actors started after all threads crashed, so all threads do readClocks. I could just set UseClocks.ALWAYS in case of system crash possible, but it makes all invocations totally different, so a lot of memory consumed in stress mode.